flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/delegation/PlannerContext.java

```diff
@@ -187,7 +187,7 @@ public FlinkCalciteCatalogReader createCatalogReader(boolean lenientCaseSensitiv
         }
 
         final SqlParser.Config newSqlParserConfig =
-                SqlParser.configBuilder(sqlParserConfig).setCaseSensitive(caseSensitive).build();
+                SqlParser.config().withCaseSensitive(caseSensitive);
```

Comment:
Is this still the same behavior? How does the `newSqlParserConfig` adopt the settings from `sqlParserConfig` above? Don't we loose information?

Reply:
good catch, thanks  🤦
i replaced it like 
```java
 final SqlParser.Config newSqlParserConfig =
                sqlParserConfig.withCaseSensitive(caseSensitive);
```